packages/modeling/src/operations/extrusions/extrudeFromSlices.d.ts

```diff
@@ -0,0 +1,14 @@
+import Geom3 from '../../geometries/geom3/type'
+import RecursiveArray from '../../utils/recursiveArray'
+
+import Slice from './slice/type'
+
+export default extrudeFromSlices
+
+export interface ExtrudeFromSlicesOptions<Base> {
```

Comment:
this is a "generic type", which you can think of as a type that takes a type as an argument (similar to how a function takes a value or reference as an argument). in this case, `Base` is a generic type for the `base` variable used in the callback. it's generic because it's up to the caller to decide what the base type is, or at least that's my understanding, since it doesn't seem to matter what you pass in as long as eventually the callback returns slices.

i will say, `extrudeFromSlices` is my new favorite learning about jscad, which i learned a month ago or so. :smiley_cat: